### ISSUE TITLE
frame_object_diff - bitset seriaziation issue (Trac #957)

### ISSUE THREAD

**Issue by icecube-trac** [icecube-trac/tix4#951]
cweaver: Should t.count() be t.size()? http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp#L104 

Very likely.  Generate a unit test that breaks because of this, then fix it.

<details>
 <summary><em>Migrated from <a href="https://code.icecube.wisc.edu/projects/icecube/ticket/957">https://code.icecube.wisc.edu/projects/icecube/ticket/957</a>, reported by david.schultzand owned by cweaver</em></summary>
<p>

```json
{
    "status": "closed",
    "changetime": "2015-07-07T22:42:37",
    "_ts": "1436308957786183",
    "description": "cweaver: Should t.count() be t.size()? http://code.icecube.wisc.edu/projects/icecube/browser/IceCube/projects/frame_object_diff/trunk/private/bitset.hpp#L104 \n\nVery likely.  Generate a unit test that breaks because of this, then fix it.",
    "reporter": "david.schultz",
    "cc": "david.schultz",
    "resolution": "fixed",
    "time": "2015-05-01T03:37:45",
    "component": "combo reconstruction",
    "summary": "frame_object_diff - bitset seriaziation issue",
    "priority": "major",
    "keywords": "bitset",
    "milestone": "",
    "owner": "cweaver",
    "type": "defect"
}
```

</p>
</details>

**Comment by icecube-trac** [comment]
Trac update at 20150519T21:50:05: 
* david.schultz changed priority from "critical" to "major"
* david.schultz changed milestone from "" to "offline V15-05-00"

**Comment by icecube-trac** [comment]
Trac update at 20150519T22:29:21: 
* cweaver changed resolution from "" to "fixed"
* cweaver changed status from "new" to "closed"
* cweaver commented:

> In changeset 132689:
> 
> ```text
> #!CommitTicketReference repository="IceCube" revision="132689"
> Add more tests to cover dynamic bitsets. 
> Fix a bug in dynamic bitset sizes. Closes #951.
> ```

**Comment by icecube-trac** [comment]
Trac update at 20150707T22:42:37: 
* olivas changed milestone from "offline V15-05-00" to ""
* olivas commented:

> Milestone offline V15-05-00 deleted
